### PR TITLE
Bracket IPv6 addresses in the HOST header (#3304)

### PR DESCRIPTION
**Pull request by pollydrag**
## What do these changes do?

Bracket IPv6 addresses in the HOST header

## Are there changes in behavior for the user?

No changes

## Related issue number

#3304

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

**Comment by webknjaz** [comment]
(@pollydrag this means that you need to add a file to `CHANGES/` folder)

**Comment by webknjaz** [comment]
And feel free to add yourself to the contributors list!

**Comment by asvetlov** [comment]
http://docs.aiohttp.org/en/stable/contributing.html#changelog-update

**Comment by pollydrag** [comment]
@webknjaz @asvetlov Change-note done. Sorry for delay

**Comment by asvetlov** [comment]
thanks!

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -162,6 +162,7 @@ Paulius Šileikis
 Paulus Schoutsen
 Pavel Kamaev
 Pawel Miech
+Pavel Polyakov
```
[the remark this answers is not in the document]
Comment (pollydrag) [reply CONTRIBUTORS.txt:165]: My bad, sorry

[the remark this answers is not in the document]
Comment (webknjaz) [reply CONTRIBUTORS.txt:165]: We probably need to lint that :)

**Pull request merged**